test/sim/moves/agility-swords-dance.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const assert = require('./../../assert');
+const common = require('./../../common');
+
+let battle;
+
+describe('Agility + Swords Dance', function () {
+	afterEach(function () {
+		battle.destroy();
+	});
+
+	it('[Gen 1] Agility should negate the paralysis speed drop.', function () {
```

Comment:
It looks like this test is failing - can you investigate?

Reply:
I think it's cus of the format choice? Not sure. I asked Kris for help with fixing it, because I really, really, really don't know how this stuff works. I know I've said this a lot of times but I'm really not a programmer...